e2e/test/scenarios/actions/reproductions/32974-linked-parameter-via-url-does-not-work.cy.spec.js

```diff
@@ -0,0 +1,170 @@
+import {
+  restore,
+  modal,
+  createAction,
+  updateDashboardCards,
+  setActionsEnabledForDB,
+  undoToast,
+  getDashboardCard,
+} from "e2e/support/helpers";
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import {
+  createMockActionParameter,
+  createMockParameter,
+} from "metabase-types/api/mocks";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const TEST_TABLE = "PRODUCTS";
+
+const ID_ACTION_PARAMETER = createMockActionParameter({
+  id: "86981cc2-2589-44b5-b559-2c8bbf5bb36a",
+  name: "ID",
+  slug: "id",
+  type: "number/=",
+  target: ["variable", ["template-tag", "id"]],
+});
+
+const ID_DASHBOARD_PARAMETER = createMockParameter({
+  name: "ID",
+  slug: "id",
+  id: "9da7bdd3",
+  type: "id",
+  sectionId: "id",
+});
+
+const DASHBOARD_DETAILS = {
+  name: "action dashboard",
+  parameters: [ID_DASHBOARD_PARAMETER],
+};
+
+const MODEL_DETAILS = {
+  name: "Products model",
+  query: { "source-table": PRODUCTS_ID },
+  database: SAMPLE_DB_ID,
+  dataset: true,
+};
+
+const QUERY_ACTION = {
```

Comment:
I try to replicate the data as close as possible when we create an action via API, so we don't have to create actions through the UI which would slow down the test.